### DISCUSSION TITLE
SELinux changed to permissive by default

### DISCUSSION THREAD

**Discussion by raghupatijha** [amazonlinux/amazon-linux-2022#180] [Q&A] [answered]
Any idea what was the reasoning behind changing SE Linux to permissive instead of Enforce by default

**Comment by stewartsmith** [comment, marked as answer]
It's pretty much what you'd guess: in the Preview time, based on feedback, a significant enough number of customers weren't ready for it to be the default. So we decided to switch to being permissive by default, but to make it easy to enable enforcing mode, and look at AL2024 as a vehicle to switch the default to Enforcing.

**Reply by raghupatijha** [reply]
Thanks Stewart

**Reply by ghost** [reply]
There is something to consider about SELinux enforcing mode by default: not all applications have properly configured SELinux configuration ruleset. This always was leading to switching SELinux into permissive mode over last decade. To allow custom applications packages to work in SELinux environment that consequence was always a case.

**Comment by mcgxtr** [comment]
SELinux is more permissive, more adaptable, and has no strict rules.
